tree.rb

```diff
@@ -19,11 +21,35 @@ def includes?(word)
   end
 
   def list
+    # %w[cat cop cup can call chat chart clap cost let letter console content contract].each { |w| add(w) }
     perform_list(@node, '', [])
   end
 
+  def save_to_file(filename = 'words.txt')
+    return false if list.empty?
+
+    File.open(file_path(filename), 'w') { |f| list.each { |word| f.puts(word) } }
+    true
+  rescue IOError
+    false
+  end
+
+  def load_from_file(filename = 'words.txt')
+    File.open(file_path(filename), 'r').each_line { |word| add(word) }
+    list.empty? ? false : true
```

Comment:
You can write this line without using Conditional Operator

Reply:
@quantumInverter I want to know, words were added to `list` or not. How can I check this, in 38 line?